integrationtest.py

```diff
@@ -45,13 +48,29 @@ def deploy_and_run_smoketest_pipeline(
     # our pipelines repository
     calcdir = lambda path: os.path.join(cc_pipelines_repo_dir, path)
 
-    pipeline_definition_file = calcdir('definitions/test/cc-smoketest.yaml')
     template_path = calcdir('templates')
     template_include_dir = cc_pipelines_repo_dir
     pipeline_name = 'cc-smoketest'
     job_name = 'cc-smoketest-master-head-update-job'
 
-    pipeline_definition = parse_yaml_file(pipeline_definition_file, as_snd=False)
+    pipeline_definition = parse_yaml_file(pipeline_definition_file, )
+
+    # obtain pipeline-definition from github at hardcoded location
```

Comment:
retrieve or obtain? please choose consistent wording. I'd figure it to be less adequate to use 'obtain', though.